src/service/rates.ts

```diff
@@ -6,19 +6,19 @@ import { getOrSave, Cachable } from "src/service/cache";
 import { HOUR, MINUTE } from "src/utils/TIME";
 import { duel, providerToDuel } from "./duel";
 import { DuelResult } from "src/utils/DuelResult";
-import { getCMC02Price } from "src/providers/UbeSwapGraph";
+// import { getCMC02Price } from "src/providers/UbeSwapGraph";
 import getCoinMarketCapPrice from "src/providers/CoinMarketCap";
 import { Tokens } from "./Data";
 import { allOkOrThrow } from "src/utils/Result";
 
-async function fetchCMCO2price(): Promise<DuelResult> {
-  const cmco2 = await duel(getCMC02Price(), getCoinMarketCapPrice("MCO2"));
-  return cmco2;
-}
+// async function fetchCMCO2price(): Promise<DuelResult> {
```

Comment:
always best to leave a `// TODO` comment on outcommented stuff so tooling can pick it up and we won't forget it later

Reply:
Jumping in here, this should just be removed. We no longer have cm02 in the reserve.

Reply:
Just this, or all reference to cmco2?

Reply:
> Just this, or all reference to cmco2?

All references, but actually this was already done and merged to main [here](https://github.com/mento-protocol/reserve-site/pull/91)

Reply:
That's not all references. There are several more, including this commented-out code, which is still on main

Reply:
> That's not all references. There are several more, including this commented-out code, which is still on main

My bad, I assumed all were removed. I'm thinking that actually it doesn't even matter now. We'll be redoing all of this logic from scratch so it won't be included.